source/includes/usage-examples/code-snippets/Find.java

```diff
@@ -0,0 +1,45 @@
+package usage_examples;
```

Comment:
nit: Package name isn't very idiomatic - Java programmers prefer studlyCaps over underscores: `usageExamples` would be better or `usage.examples` possibly the best.